### PR TITLE
Bitfinex WS Fixes: Reconnect and crash on previously non-traded assets/currency

### PR DESCRIPTION
**Pull request by crubb**
As the title says. Also removed some trailing white spaces.

@nedievas?

### CODE REVIEW

**Review by DeviaVir — APPROVED** [review]

File: extensions/exchanges/bitfinex/exchange.js [hunk 1]
```diff
@@ -46,13 +46,13 @@ module.exports = function container (get, set, clear) {
       }
       ws_trades.push(newTrade)
     })
-    
+
```
Comment (nedievas) [inline extensions/exchanges/bitfinex/exchange.js:49]: remove

File: extensions/exchanges/bitfinex/exchange.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ module.exports = function container (get, set, clear) {
     if (message[0] != "undefined")
```
Comment (nedievas) [inline extensions/exchanges/bitfinex/exchange.js:65]: remove at 64

File: extensions/exchanges/bitfinex/exchange.js [hunk 3]
```diff
@@ -461,22 +461,22 @@ module.exports = function container (get, set, clear) {
...
     getOrder: function (opts, cb) {
       var order = ws_orders['~' + opts.order_id]
 
-      if (order.status === 'rejected' && order.reject_reason === 'post only') {        
+      if (order.status === 'rejected' && order.reject_reason === 'post only') {
         return cb(null, order)
       } else if (order.status === 'rejected' && order.reject_reason === 'zenbot canceled') {
-        return cb(null, order)        
+        return cb(null, order)
       }
```
Comment (nedievas) [inline extensions/exchanges/bitfinex/exchange.js:481]: remove

**Review by nedievas — APPROVED** [review]
more trailing spaces to remove...I suppose this update has fix for balance bug which crashes zenbot with a fresh asset trade attempt.

**Pull request merged**